### PR TITLE
build(clint): remove old comment style check

### PR DESCRIPTION
**Pull request by lewis6991**
Use uncrustify instead.

### CODE REVIEW

**Comment by dundargoc** [comment]
Ackshually, uncrustify can't convert multiline comments to `//` which is why I haven't removed this check.

**Comment by lewis6991** [comment]
Hmm I tested it really quickly and it seemed to which is why I started the PR. Let me double check.

**Comment by zeertzjq** [comment]
It doesn't work for a large block of comment:
```diff
diff --git a/src/nvim/ex_docmd.c b/src/nvim/ex_docmd.c
index b854bdb6f..0b9c38369 100644
--- a/src/nvim/ex_docmd.c
+++ b/src/nvim/ex_docmd.c
@@ -117,9 +117,11 @@ typedef struct {
 
 #define FREE_WCMD(wcmd) xfree((wcmd)->line)
 
-/// Structure used to store info for line position in a while or for loop.
-/// This is required, because do_one_cmd() may invoke ex_function(), which
-/// reads more lines that may come from the while/for loop.
+/*
+ * Structure used to store info for line position in a while or for loop.
+ * This is required, because do_one_cmd() may invoke ex_function(), which
+ * reads more lines that may come from the while/for loop.
+ */
 struct loop_cookie {
   garray_T *lines_gap;               // growarray with line info
   int current_line;                     // last read line from growarray
```

**Comment by lewis6991** [comment]
Dam

File: src/nvim/help.c [hunk 1]
```diff
@@ -319,7 +319,7 @@ int find_help_tags(const char *arg, int *num_matches, char ***matches, bool keep
     { "[*",          "[star" },
     { "]*",          "]star" },
     { ":*",          ":star" },
-    { "/*",          "/star" },  // NOLINT
```
Comment (justinmk) [inline src/nvim/help.c:322]: Ah this would have been a nice change!

**Pull request closed without merging**